src/software/simulation/er_force_simulator.h

```diff
@@ -198,6 +202,7 @@ class ErForceSimulator
 
     amun::SimulatorSetup er_force_sim_setup;
     std::unique_ptr<camun::simulator::Simulator> er_force_sim;
+    EuclideanToWheel euclidean_four_wheel_convert;
```

Comment:
nit: to stay consistent
```suggestion
    EuclideanToWheel euclidean_to_four_wheel;
```